channels/us.m3u

```diff
@@ -845,8 +839,6 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxchive/CDN/playlist.m3u8
 https://d275e95ir6yrv9.cloudfront.net/dist/localnow/11/hls/master/playlist.m3u8
 #EXTINF:-1 tvg-id="Choppertown.us" tvg-name="Choppertown" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f9c5eb78083e72bec0d0522" group-title="Auto",Choppertown (1080p)
 https://oneworlddigital-choppertown-1.vizio.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="Choppertown.us" tvg-name="Choppertown" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f9c5eb78083e72bec0d0522" group-title="Auto",Choppertown (720p) [Offline]
-https://linear-11.frequency.stream/dist/plex/11/hls/master/playlist.m3u8
```

Comment:
Not 24/7

![image](https://user-images.githubusercontent.com/30985701/132143297-e53c0bc3-26df-4389-9105-bd497a06d6d5.png)